ci-operator/step-registry/openshift/upgrade/qe/test/cert-rotation/openshift-upgrade-qe-test-cert-rotation-commands.sh

```diff
@@ -119,6 +119,11 @@ oc adm restart-kubelet nodes --all --directive=RemoveKubeletKubeconfig
 oc adm reboot-machine-config-pool mcp/worker mcp/master
 oc adm wait-for-node-reboot nodes --all
 
+# WARNING: On some platforms this step may prevent new nodes from joining the cluster. To verify whether this step is safe to perform on your cluster, please follow this process:
+workerUserData="$(oc -n openshift-machine-api get secret/worker-user-data -o=jsonpath='{.data.userData}' | base64 -d)" || true
+workerUserDataManaged="$(oc -n openshift-machine-api get secret/worker-user-data-managed -o=jsonpath='{.data.userData}' | base64 -d)" || true
```

Comment:
@liangxia the two commands' result will be in the following format:
`{"ignition":{"config":{"merge":[{"source":"$URL"}]},"security":{"tls":{"certificateAuthorities":[{"source":"data:text/plain;charset=utf-8;base64,..."}]}},"version":"3.2.0"}}`

For the URL, if it looks like "https://api-int.$CLUSTERNAME.$DOMAIN:22623/config/worker", the cluster is safe to run the following steps. If you see the IP address directly instead, looks like https://192.168.0.1:22623/config/worker do NOT perform this step.